### ISSUE TITLE
More compare exp on different dataset or with more baseline？

### ISSUE THREAD

**Issue by snowisland1** [jhwjhw0123/Imbalance-XGBoost#6]
在我的数据实验上两种方法调参后在不平衡分类任务中，
效果都比XGBoost原模型差，并且模型训练时间变长了。
不知道这样改进的意义何在，动机是什么呢？是否经过充分论证，缺少足够实验证明改进方法的有效性吧。
陈天奇都说过，修改损失函数意义不大，AWS是不稳定的，可能对模型提升没有帮助。
--------
In my data experiment, This two methods are adjusted in the unbalanced classification task（100:1）, the evl（F-score） is worse than the original XGBoost model, and the model training time becomes longer.

I don't know what the significance of this improvement is, what is the motivation?Whether it is fully demonstrated and there is not enough experiment to prove the effectiveness of the improvement method in the readme and u paper.

Chen Tianqi said that modifying the loss function is of little significance. AWS is unstable and may not help the model improve.

![image](https://user-images.githubusercontent.com/37918417/67556711-388c4900-f746-11e9-8493-7c1dfe0825f0.png)

----》https://www.kaggle.com/c/higgs-boson/discussion/10286

**Comment by niuwan1** [comment]
Agree with @snowisland1, performance of XGBoost focal loss is larger than the XGBoost-logloss
